README.ita.md

```diff
@@ -0,0 +1,111 @@
+English|[中文](/README.chs.md)|[Italiano](/README.ita.md)
```

Comment:
No need to link to the page itself, you need to link the original English page, as following:

Italiano|[English](/README.md)